egs2/kiritan/svs1/local/extract_from_midi.py

```diff
@@ -0,0 +1,23 @@
+"""
```

Comment:
empty file?

Reply:
This piece of code may help to to extract xml input from midi if needed.
I just add it for kiritan recipe for partners temporarily, if needed.

Reply:
I would suggest to remove this one for the version merge to master. You may add anohter PR to branch `muskit_recipe` instead, for this purpose

Reply:
okay, removed it here.